tests/test_metadata.py

```diff
@@ -32,3 +32,33 @@ def test_copying_services_metadata():
 
             for question in copying_services_data['questions_to_copy']:
                 assert question in names_ids_fields
+
+
+def test_g11_service_questions_to_scan_for_bad_words_metadata():
+    """
+    Ensure every key listed under service_questions_to_scan_for_bad_words.service_questions points
+    to a valid question file, id or field.
+    """
+    service_question_filenames = glob.glob(
+        'frameworks/g-cloud-11/metadata/service_questions_to_scan_for_bad_words.yml'
+    )
+    assert len(service_question_filenames) > 0
+    for service_question_filename in service_question_filenames:
+
+        with open(service_question_filename) as copying_services_file:
+            copying_services_data = yaml.safe_load(copying_services_file.read())
+
+            names_ids_fields = set()
+            for filename in os.listdir(f'frameworks/g-cloud-11/questions/services'):
+                if filename.startswith('multiq'):
+                    continue
+                with open(f'frameworks/g-cloud-11/questions/services/{filename}') as f:
+                    contents = yaml.safe_load(f)
+                    if contents.get('id'):
+                        names_ids_fields.add(contents['id'])
+                    if contents.get('fields'):
+                        names_ids_fields.update([v for v in contents.get('fields').values()])
+                    names_ids_fields.add(filename[:-4])
+
+            for question in copying_services_data['service_questions']:
```

Comment:
Is this for loop meant to be in the `for service_question_filename` loop?

Reply:
Some c+p from the test above has sneaked in. Will push a commit to fix.